BaseCollector.py

```diff
@@ -132,10 +132,14 @@ def wait_for_inventory_data(self):
         return
 
     def create_http_response_metric(self, target, token, collector):
-        http_response_code = Vrops.get_http_response_code(target, token)
-        http_gauge = GaugeMetricFamily('vrops_http_response_code', 'vrops-exporter', labels=['target', 'class'])
-        http_gauge.add_metric(labels=[self.target, collector.lower()], value=http_response_code)
-        return http_response_code, http_gauge
+        api_responding = Vrops.get_http_response_code(target, token)
+        gauge = GaugeMetricFamily('vrops_api_response', 'vrops-exporter', labels=['target', 'class'])
+        gauge.add_metric(labels=[self.target, collector.lower()], value=api_responding)
+
+        if api_responding > 200:
```

Comment:
if `get_http_response_code` would run into the try except Exception, it would return 0, thus it won't match this condition. Shouldn't we check if there is a return value, if it is, return it, otherwise 500 (for the exception case)?

Reply:
We could choose between `404 Not Found` or `408 Request Timeout`

Reply:
Why not 500? We are catching a random exception there. This is neither a timeout nor 404.

Reply:
The exporter application is the client in this scenario. So 4** HTTP codes are the right ones. If vrops not reachable:`Problem connecting to {target} - Error: {e}` I would return `408 Request Timeout`. `500` would mean, the server has an internal error. But we do not know, there is no response.

Reply:
But when I look at server HTTP codes, we could use `503 Service Unavailable`. I admit, this is maybe better.

Reply:
I'd take your 503 ;)